BraintreeUIKit/Components/BTUIKExpiryInputView.m

```diff
@@ -30,12 +30,12 @@ - (instancetype)initWithFrame:(CGRect)frame {
         self.months = @[@"01", @"02", @"03", @"04", @"05", @"06", @"07", @"08", @"09", @"10", @"11", @"12"];
         
         NSDate *currentDate = [NSDate date];
-        NSCalendar* calendar = [NSCalendar currentCalendar];
-        NSDateComponents* components = [calendar components:NSCalendarUnitYear|NSCalendarUnitMonth fromDate:currentDate];
+        NSCalendar *calendar = [NSCalendar calendarWithIdentifier:NSCalendarIdentifierGregorian];
```

Comment:
Don't know much about the code ramifications here, is this a breaking change?

Reply:
No, this shouldn't be a breaking change (checked in with @billwerges on this)